LauncherInjector/main.cpp

```diff
@@ -224,6 +225,15 @@ bool LoadNorthstar()
 	}
 
 	((bool (*)())Hook_Init)();
+
+
+	Hook_Init = GetProcAddress(hHookModule, "LoadPlugins");
+	if (!hHookModule || Hook_Init == nullptr)
+	{
+		LibraryLoadError(GetLastError(), L"Plugins.dll", buffer);
+		return false;
```

Comment:
You don't want to use `LibraryLoadError` since it'll lead to confusing error messages. Notice how I handled errors for the stubs.